index.js

```diff
@@ -625,6 +625,19 @@ var commands = {
 	}
 };
 
+// log a debug message only if its category is enabled
+// first argument is category, following arguments are console.log's input
+var debugMsg = function() {
+	var category = arguments[0];
+	if (category in gd.debug) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["gd" is not defined.](https://www.codacy.com/app/vpzomtrrfrt/turbo-spork-server/file/2647138125/issues/source?bid=3248399&fileBranchId=3323017#l632)